libs/wingc/src/type_check.rs

```diff
@@ -3337,18 +3367,9 @@ impl<'a> TypeChecker<'a> {
 				}
 
 				let instance_type = self.type_check_exp(object, env);
-
-				// TODO Use error type instead of "anything" here https://github.com/winglang/wing/issues/884
-				if instance_type.is_anything() {
-					// Check to see if this reference is actually an invalid usage of a namespace
-					if let Some(ref_udt) = self.reference_to_udt(reference) {
-						let lookup = self.resolve_user_defined_type(&ref_udt, env, self.statement_idx);
-						if let Err(t) = lookup {
-							if t.message.ends_with("to be a type but it's a namespace") {
-								return self.make_error_variable_info(false);
-							}
-						}
-					}
+				// If resolving the object's type failed, we can't resolve the property either
+				if instance_type.is_error() {
```

Comment:
For the vscpde autocomplete to work we need to make sure `resolve_user_defined_type` is called here to eagerly import the types inside a namespace in case this reference only points to a name space (and not to an actual type). Fix!